hazelcast-spring/src/test/resources/log4j2-info.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright (c) 2008-2017, Hazelcast, Inc. All Rights Reserved.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~ http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<Configuration status="ERROR">
```

Comment:
log4j2-info.xml is for what?

Reply:
It's used by the PR builders to reduce the amount of log output.
```
-Dlog4j.configurationFile=log4j2-info.xml
```

Reply:
ah ok 👍

Reply:
There is no specific reason for it. `hazelcast` and `hazelcast-client` modules have this file, so I applied same structure here.